runtime/src/lib.rs

```diff
@@ -1292,19 +1203,14 @@ construct_runtime! {
         Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 29,
         Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 30,
         Bounties: pallet_bounties::{Pallet, Call, Storage, Event<T>} = 31,
-        Lottery: pallet_lottery::{Pallet, Call, Storage, Event<T>} = 32,
-        Gilt: pallet_gilt::{Pallet, Call, Storage, Event<T>, Config} = 33,
         // Pallets
         OrmlVesting: orml_vesting::{Pallet, Storage, Call, Event<T>, Config<T>} = 34,
         Currencies: orml_currencies::{Pallet, Call, Event<T>} = 35,
         Tokens: orml_tokens::{Pallet, Storage, Event<T>, Config<T>} = 36,
         PolkadexFungibleAsset: polkadex_fungible_assets::{Pallet, Call, Storage, Event<T>} = 37,
```

Comment:
Remove Polkadex FungibleAssets, SubstrateeRegistery, PolkadexOcex. IDO and Polkapool, These pallets will not be there for mainnet launch and will be upgraded via Runtime

Reply:
Our focus will be on ERC20 PDEX migration and making sure staking and treasury are set and validators secure the network.

Reply:
@Gauthamastro  do you know why we added this enumeration last week?

Reply:
This PR is on develop. There will be a cherry-pick to main later

Reply:
Alright